gitpod-io/website
# New announement banner for Kubecon EU

## Description



|Before|After|
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/55068936/226530896-4471d65a-fd11-4322-9117-6bb391f7971a.png)|![image](https://user-images.githubusercontent.com/55068936/226530988-8f9fcc8d-6e8d-43e7-a76f-e17d08b0fe95.png)|

## Related Issue(s)



Fixes #3522

## How to test



- Open Preview and see banner at top of the navbar 👀 



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3528"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
